vector/src/main/res/layout/dialog_change_password.xml

```diff
@@ -8,24 +9,51 @@
     android:paddingEnd="?dialogPreferredPadding"
     android:paddingRight="?dialogPreferredPadding">
 
-    <EditText
-        android:id="@+id/change_password_old_pwd_text"
+    <android.support.design.widget.TextInputLayout
         android:layout_width="match_parent"
-        android:layout_height="wrap_content"
-        android:hint="@string/settings_old_password"
-        android:inputType="textPassword" />
+        android:layout_height="wrap_content">
+
+        <android.support.design.widget.TextInputEditText
+            android:id="@+id/change_password_old_pwd_text"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:hint="@string/settings_old_password"
+            android:inputType="textPassword" />
+    </android.support.design.widget.TextInputLayout>
 
-    <EditText
-        android:id="@+id/change_password_new_pwd_text"
+    <android.support.design.widget.TextInputLayout
         android:layout_width="match_parent"
-        android:layout_height="wrap_content"
-        android:hint="@string/settings_new_password"
-        android:inputType="textPassword" />
+        android:layout_height="wrap_content">
 
-    <EditText
-        android:id="@+id/change_password_confirm_new_pwd_text"
+        <android.support.design.widget.TextInputEditText
+            android:id="@+id/change_password_new_pwd_text"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:hint="@string/settings_new_password"
+            android:inputType="textPassword"
+            />
```

Comment:
Please format the file and rearrange entry.

Reply:
didn't get it. Rearrange entry ?

Reply:
In Android Studio, menu "Code/Show Reformat File Dialog", you can check "Rearrange code".

Reply:
I did . Nothing changed  "No lines changed"